source/include/rfmulti.f90

```diff
@@ -30,9 +30,10 @@
 
   Sp1 = Sp1 * (x_t+imag*y_t);
 
-  yv1(j) = yv1(j) - ((real(Cp0)*c1e3)*moidpsv(j))
-  yv2(j) = yv2(j) + ((aimag(Cp0)*c1e3)*moidpsv(j))
-  ejv(j) = ejv(j) - ((real(Sp1)*(c1e3*(e0f*(crabfreq*(two*pi))))))/clight
+  yv1(j) = yv1(j) - (((REAL(Cp0))*tiltc(i))  + ((AIMAG(Cp0)))*tilts(i))*(moidpsv(j)*c1e3)  
+  yv2(j) = yv2(j) + (((-REAL(Cp0))*tilts(i)) + ((AIMAG(Cp0)))*tiltc(i))*(moidpsv(j)*c1e3)
+  ejv(j)   = ejv(j) - ((real(Sp1)*(c1e3*(e0f*(crabfreq*(two*pi))))))/clight
+  ejf0v(j) = ejfv(j)
```

Comment:
The line `ejf0v(j) = ejfv(j)` is redundant. This is already handled in `part_updatePartEnergy` when using `updateAngles = .true.`